lib/api/Cargo.toml

```diff
@@ -27,6 +27,8 @@ wat = { version = "1.0", optional = true }
 thiserror = "1.0"
 more-asserts = "0.2"
 target-lexicon = { version = "0.11", default-features = false }
+loupe = { path = "../../../loupe/crates/loupe" }
+loupe-derive = { path = "../../../loupe/crates/loupe-derive" }
```

Comment:
Lets publish the crate before merging the PR.
Having this comment as a reminder :)

Reply:
Absolutely. It's a WIP, I'm modifying `loupe` quite a lot for the moment. As soon as we finish this project, I will release `loupe` and update this PR.